auto_nag/topcrash.py

```diff
@@ -359,6 +361,7 @@ def __get_params_from_criterion(self, criterion: dict):
         params = {
             "product": criterion["product"],
             "release_channel": criterion["channel"],
+            "major_version": self._get_major_version_constrain(criterion["channel"]),
```

Comment:
That's useful, I didn't know it existed!

Reply:
Me too, I found it when I was trying to experiment with Sebastian's suggestion :)